<launch>

  <arg name="base" default="$(optenv TURTLEBOT_BASE kobuki)"/>
  <arg name="battery" default="$(optenv TURTLEBOT_BATTERY /proc/acpi/battery/BAT0)"/>
  <arg name="stacks" default="$(optenv TURTLEBOT_STACKS hexagons)"/>
  <arg name="3d_sensor" default="$(optenv TURTLEBOT_3D_SENSOR kinect)"/>

  <arg name="model" default="$(find turtlebot_description)/robots/kobuki_hexagons_kinect.urdf.xacro" />

   <param name="robot_description" command="$(find xacro)/xacro.py '$(arg model)'" />

  <!-- Bring up robot1 -->
  <group ns="turtlebot_1">
<param name="tf_prefix" value="turtlebot_1" />
    <include file="$(find lucia_sim_2014)/launch/robot.launch.xml" >
      <arg name="robot_name"  value="turtlebot_1" />
      <arg name="x" value="0"/>
      <arg name="y" value="0"/>
    </include>
  </group>

  <!-- Bring up robot2 -->
  <group ns="turtlebot_2">
<param name="tf_prefix" value="turtlebot_2" />
    <include file="$(find lucia_sim_2014)/launch/robot.launch.xml" >
      <arg name="robot_name"  value="turtlebot_2" />
      <arg name="x" value="-1"/>
      <arg name="y" value="-1"/>
    </include>
  </group>

  <!-- Bring up robot3 -->
  <group ns="turtlebot_3">
<param name="tf_prefix" value="turtlebot_3" />
    <include file="$(find lucia_sim_2014)/launch/robot.launch.xml" >
      <arg name="robot_name"  value="turtlebot_3" />
      <arg name="x" value="2"/>
      <arg name="y" value="2"/>
    </include>
  </group>

  <!-- Bring up robot4 -->
  <group ns="turtlebot_4">
<param name="tf_prefix" value="turtlebot_4" />
    <include file="$(find lucia_sim_2014)/launch/robot.launch.xml" >
      <arg name="robot_name"  value="turtlebot_4" />
      <arg name="x" value= "1"/>
      <arg name="y" value="-1"/>
    </include>
  </group>


</launch>
